docs/Installing.asciidoc

```diff
@@ -411,10 +411,12 @@ connecting to a remote PostgreSQL database with a username and password:
 [source, ini]
 --------------------------------------------------------------------------------
 [production]
-dsn = dbi:Pg:dbname=openqa;host=db.example.org
+dsn = dbi:Pg:dbname=openqa
```

Comment:
Two problems here. One, the example no longer matches the text, which says "Here is an example for
connecting to a remote PostgreSQL database with a username and password:". Two, I do think it's useful to have the correct syntax for a remote host here so people don't have to go look it up (which is actually quite *hard*, due to how the perl modules for database interactions are laid out and documented...)